content/python/concepts/pip/pip.md

```diff
@@ -0,0 +1,50 @@
+---
+Title: 'pip'
+Description: 'Much like how npm is used to manage code packages in JavaScript, pip also offers practical ways to install and use pre-made code.'
+Tags:
+  - 'Conceptual'
+  - 'Packages'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/learn-python3'
+---
+
+`pip` is the package installer used in Python to install code packages. A similar tool is Node Package Manager (npm) in JavaScript. Much like npm, `pip` is the same community driven package space, but for Python.
+
+## Packages
+
+The concept of packages is not unique to `pip`. A package is essentially a collection of code that is designed for a specific purpose. One example is the [neuralart](https://pypi.org/project/neuralart/) project, which is a project on Python Package Index (PyPI) that is used for "rendering generative art from a randomly initialized neural network." It's a very cool package that creates computer generated art.
+
+## Python Package Index Repository (PyPI)
+
+[The Python Package Index Repository](https://pypi.org/), otherwise called PyPI, hosts a large collection of these packages. The primary tool used to interact with PyPI is the command line, specifically `pip`. Packages on this network typically offer a list of requirements to run the package, how to install it, as well as examples of the package being implemented
```

Comment:
I think the abbreviation, `PyPI`, could be moved next to the expanded equivalent and enclosed in parenthesis. LIke so,

```
The [Python Package Index Repository (or **PyPI**)](https://pypi.org/) hosts a large collection...
```